.github/workflows/publish.yml

```diff
@@ -22,3 +22,5 @@ jobs:
     - run: npm publish
       env:
         NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        AWS_ACCESS_KEY: ${{ secrets.AWS_ACCESS_KEY }}
+        AWS_SECRET_KEY: ${{ secrets.AWS_SECRET_KEY }}
```

Comment:
```suggestion
```

Reply:
We need this env var, it's not a duplicate of the one above 😅

Reply:
wow my eyes and or brain 😅